### PR TITLE
Lore and asset tracking

### PR DESCRIPTION
**Pull request by C-Shart**
Going ahead and submitting a PR now so I can just make a new branch off main for the code updates

### CODE REVIEW

File: beetle_tester.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import arcade
+from beetles import Beetle
+from teams import Team
+from team_color import TeamColor
+
+# Constants
+SCREEN_WIDTH = 1280
+SCREEN_HEIGHT = 720
+SCREEN_TITLE = "Welcome to the Ouch, Motherfucker"
```
Comment (chimerror) [inline beetle_tester.py:9]: We should probably give the tester a different title.

File: beetle_tester.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        arcade.set_background_color(arcade.color.WHITE)
+
+    def setup(self):
+        self.background = arcade.load_texture("Assets/Images/octagon.png")
+
+        self.green_team = Team(TeamColor.GREEN, 320, 260)
+        self.green_team.set_up_team()
+
+        self.red_team = Team(TeamColor.RED, 960, 260)
+        self.red_team.set_up_team()
```
Comment (chimerror) [inline beetle_tester.py:30]: Might be worth adding a TODO that we need to replace this with the tester set-up, which will likely be different from the game set-up.

**Review by chimerror — APPROVED** [review]
This looks fine to me. I will note that we should probably move the Awareness diagram CSP project to Notes/Art since that's where I put my notes CSP project.

Other than that, this looks fine, so I'm marking approve. I'll wait to merge until you respond, though.

File: beetle_tester.py (hunk 1 above)
Comment (C-Shart) [reply beetle_tester.py:9]: done

**Pull request merged**